### DISCUSSION TITLE
How to set archive flag, or get archived chunks to revert to standard chunks

### DISCUSSION THREAD

**Discussion by VideoFX** [moosefs/moosefs#469] [General]
I found that once chunks enter the archived state, they cannot go back to standard chunks.

What I tried:
I modified the file. However modifications are ignored by the archiving system it seems. 
I manually pushed a "mfsscadmin" modify command and "mfssetsclass" to the file/directory to see if that would refresh it's state. I pushed the same storage class, It had no effect. 

The storage class I'm using is:
mfsscadmin /mnt/moosefs/ modify SSD_to_HD -C U,X -K U,X -A BH,AH -d 1 -m D

I also tried using:
mfssetsclass -r SSD_to_HD     /mnt/moosefs/<files>

So archiving will work, it will send a file to the archive. But once its there, it seems like there is no way to reverse that, and get it back to a standard/keep state. Is there a command I can use to clear the archive flag?

Thank you. Cheers!

**Comment by xandrus** [comment]
Hi @VideoFX,

Yes, there is a command that can clear archive flag for specific file.
Please check `mfsclrarchive` command.

**Reply by VideoFX** [reply]
This helps a ton. Thank you!
